docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/biography/node--cgov-biography--full.html.twig

```diff
@@ -68,15 +68,9 @@
 
         <div id="cgvBody">
           {{ content.body }}
-
-          {# TODO: replace placeholder below with customizable content #}
-          <div class="press-contact">
-            <h2>Press Contacts</h2>
-            <p>To request an interview with NCI researchers, contact the NCI Office of Media Relations.<br />
-              <a href="mailto:ncipressofficers@mail.nih.gov" class="press-contact-email">ncipressofficers@mail.nih.gov</a><br />
-              <span class="press-contact-phone">240-760-6600</span>
-            </p>
-          </div>
+          {% if content.field_bio_press_info[0]['#markup'] == "On" %}
```

Comment:
this could just be the following?
`content_field_bio_press_info | field_value`

0 should be false and anything not zero is true.

Reply:
+1